src/pages/MonitorLogPage.jsx

```diff
@@ -31,12 +31,12 @@ import { parseISO } from 'date-fns';
 import PropTypes from 'prop-types';
 import { useUserContext } from '../common/UserContext/UserContext';
 import { OCHBackend } from '../common/utils';
-import AdditionalSpeciesTab from '../components/MonitorLog/AdditionalSpeciesTab';
-import GeneralInfoTab from '../components/MonitorLog/GeneralInfoTab';
-import HumanActivity from '../components/MonitorLog/HumanActivityTab';
-import ListedSpeciesTab from '../components/MonitorLog/ListedSpeciesTab';
-import PredatorsTab from '../components/MonitorLog/PredatorsTab';
-import ReviewSubmitTab from '../components/MonitorLog/ReviewSubmitTab';
+import AdditionalSpeciesTab from '../components/monitorLog/AdditionalSpeciesTab';
+import GeneralInfoTab from '../components/monitorLog/GeneralInfoTab';
+import HumanActivity from '../components/monitorLog/HumanActivityTab';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../components/monitorLog/HumanActivityTab"